src/client/meshgen/collector.cpp

```diff
@@ -0,0 +1,119 @@
+#include "collector.h"
+#include "log.h"
+#include "mesh.h"
+
+void MeshCollector::append(const TileSpec &tile, const video::S3DVertex *vertices,
+		u32 numVertices, const u16 *indices, u32 numIndices)
+{
+	for (int layernum = 0; layernum < MAX_TILE_LAYERS; layernum++) {
```

Comment:
i think a c++11 for range based loop works here no ?

Reply:
`layernum` is used in the loop.